docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -0,0 +1,167 @@
+        src="/static/images/creating-google-service-account/fcm-v1/new-service-account/04-upload-credential-3.png"
+        style={{ maxWidth: 800 }}
+      />
+    </Tab>
+  </Tabs>
+</Step>
+
+<Step label="5">
+  You're all set! You can now send notifications to Android devices via Expo Push Notifications using the FCM V1 protocol.
+
+  <ImageSpotlight
+    alt="Service account key successfully uploaded"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/05-upload-credential-complete.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+## Use an existing Google Service Account Key
+
+<Step label="1">
+  Open the [IAM Admin page](https://console.cloud.google.com/iam-admin/iam?authuser=0) in Google Cloud Console. In the Permissions tab, locate the **Principal** you intend to modify and click the pencil icon for **Edit Principal**.
+
+  <ImageSpotlight
+    alt="Edit Principal in IAM Admin page in Google Cloud Console"
+    src="/static/images/creating-google-service-account/fcm-v1/existing-service-account/01-iam-admin-page.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="2">
+  Click **Add Role** and select the **Firebase Messaging API Admin** role from the dropdown. Click **Save**.
+
+  <ImageSpotlight
+    alt="Assign roles"
+    src="/static/images/creating-google-service-account/fcm-v1/existing-service-account/02-add-role-1.png"
+    style={{ maxWidth: 800 }}
+  />
+  <ImageSpotlight
+    alt="Select the Firebase Messaging API Admin role"
+    src="/static/images/creating-google-service-account/fcm-v1/existing-service-account/02-add-role-2.png"
+    style={{ maxWidth: 800 }}
+  />
+  <ImageSpotlight
+    alt="Save the role assignment"
+    src="/static/images/creating-google-service-account/fcm-v1/existing-service-account/02-add-role-3.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="3">
+  You have to specify to EAS which JSON credential file to use for sending FCM V1 notifications, using EAS CLI or in [Expo dashboard](https://expo.dev). You can upload a new JSON file or select a previously uploaded file.
+
+  <Tabs tabs={["EAS CLI", "expo.dev"]}>
+    <Tab>
+      - Run `eas credentials`
+      - Select `Android` > `production` > `Google Service Account`
+      - Select `Manage your Google Service Account Key for Push Notifications (FCM V1)`
+    </Tab>
+    <Tab>
+      - - Under **Project settings**, click [Credentials](https://expo.dev/accounts/[account]/projects/[project]/credentials) in the navigation menu
+      - For **Android**, click **Add Application Identifier** or select an existing **Application identifier**
+         - Under **Service Credentials**, find **FCM V1 service account key**
+       <ImageSpotlight
+	        alt="Project credentials page on expo.dev"
+	        src="/static/images/creating-google-service-account/fcm-v1/new-service-account/04-upload-credential-1.png"
+	        style={{ maxWidth: 800 }}
+      />
```

Comment:
```suggestion
      - For **Android**, click **Add Application Identifier** or select an existing **Application identifier**
      - Under **Service Credentials**, find **FCM V1 service account key**

      <ImageSpotlight
        alt="Project credentials page on expo.dev"
        src="/static/images/creating-google-service-account/fcm-v1/new-service-account/04-upload-credential-1.png"
        style={{ maxWidth: 800 }}
      />
```

This will fix the CI failure. It was caused by my suggestion here earlier.